modules/constants/src/webgl-types.ts

```diff
@@ -364,7 +368,7 @@ export type GLFunctionParameters = {
   colorMask?: [boolean, boolean, boolean, boolean] | boolean[];
 
   cull?: boolean;
-  cullFace?: GL.FRONT | GL.BACK | GL.FRONT_AND_BACK;
+  cullFace?: GLCullFaceMode;
```

Comment:
Oh, that makes sense! I've removed the intermediate types here, so the exposed APIs in this file are explicit. I have kept some aliases above, so that `device-parameters.ts` has a source of truth to typecheck against for misspellings and such.